src/API/Reports/Import/Controller.php

```diff
@@ -259,24 +259,21 @@ public function cancel_import( $request ) {
 	 * @return WP_Error|WP_REST_Response
 	 */
 	public function delete_imported_items( $request ) {
-		$delete = ReportsSync::delete_report_data();
+		$result = array(
+			'status'  => 'success',
+			'message' => ReportsSync::delete_report_data(),
+		);
 
-		if ( is_wp_error( $delete ) ) {
-			$result = array(
-				'status'  => 'error',
-				'message' => $delete->get_error_message(),
-			);
-		} else {
-			$result = array(
-				'status'  => 'success',
-				'message' => $delete,
-			);
+		if ( is_wp_error( $result['message'] ) ) {
+			$result['status']  = 'error';
+			$result['message'] = $result['message']->get_error_message();
```

Comment:
Mutating `$result['message']` to avoid a variable seems off to me, I would rather have this explicit